unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -473,6 +473,20 @@ public void SetEntityParent(string entityId, string parentId)
                 return;
             }
 
+            if (me != null && parentId == "PlayerEntity")
```

Comment:
As this is a performance critical path, maybe we can group the `me != null` checks? Any small performance saving here is gold.